app/socket-apis/get-points-of-ids.js

```diff
@@ -0,0 +1,44 @@
+// https://github.com/EnCiv/civil-pursuit/issues/203
+const Points = require('../models/points')
+const { ObjectId } = require('mongodb')
+
+async function getPointsOfIds(ids, callback, currentUserId) {
+  try {
```

Comment:
please see get-points-for-round for how to get the synuser.id, and validate that this is a logged in users.  
Also, please follow that pattern for verifying arguments.